static-website-aws-csharp/Program.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using Pulumi;
+using Aws = Pulumi.Aws;
+using SyncedFolder = Pulumi.SyncedFolder;
+
+return await Deployment.RunAsync(() => 
+{
+    var config = new Config();
+    var awsRegion = config.Get("awsRegion") ?? "us-west-2";
```

Comment:
Should these be using the `aws:region` config?

Reply:
Indeed — sure what happened there, maybe something amiss with `pulumi convert`. Will take a look!

Reply:
Fixed!